README.md

```diff
@@ -5,41 +5,66 @@ This repo allows you to deploy a new instance of the victims-web server on opens
 
 ## Running on OpenShift
 ### Prerequisites
-1. You have a valid account with OpenShift
-2. You have followed the instructions at https://www.openshift.com/get-started
-3. You have *rhc* installed and ready
+1. You have a valid account with OpenShift https://www.openshift.com/get-started
+2. You have the s2i tool installed locally https://github.com/openshift/source-to-image
 
-### Creating the app
-#### One Shot Deployment
-This is pretty straight forward, run the following command. The app should be deployed to ```http://victims-NAMESPACE.rhcloud.com```. See bottom for a sample output.
+### Log in to Openshift
 ```sh
-rhc app create victims mongodb-2.2 rockmongo-1.1 python-2.7 --from-code git://github.com/victims/victims-server-openshift.git
+oc login https://console.starter-us-east-1.openshift.com
```

Comment:
This should probably say ``https://youropenshiftinstance.example.com`` or something like that.